sovrin_client/test/cli/test_nym_suspension.py

```diff
@@ -105,7 +105,7 @@ def testTrusteeSuspendingTrustAnchor(be, do, trusteeCli, trustAnchorAdded,
        expect=nymAddedOut, mapper=trustAnchorAdded)
     s = SimpleSigner().identifier
     be(trustAnchorCli)
-    errorMsg = 'UnauthorizedClientRequest'
+    errorMsg = "UnauthorizedClientRequest('USER cannot add USER'"
```

Comment:
Changed USER to 'None role' and added TODO comment. We need to come back to this once now an answer